system-parachains/asset-hubs/asset-hub-kusama/src/lib.rs

```diff
@@ -1071,14 +1071,9 @@ pub type SignedExtra = (
 pub type UncheckedExtrinsic =
 	generic::UncheckedExtrinsic<Address, RuntimeCall, Signature, SignedExtra>;
 
-parameter_types! {
-	pub DmpQueueName: &'static str = "DmpQueue";
-}
-
 /// Migrations to apply on runtime upgrade.
 pub type Migrations = (
-	frame_support::migrations::RemovePallet<DmpQueueName, RocksDbWeight>,
```

Comment:
I think we should be able now to remove `cumulus-pallet-dmp-queue` dependency from everywhere (at least AHK/BHK uses that and/or maybe also remove it from the main Cargo.toml).